torch/storage.py

```diff
@@ -968,69 +968,69 @@ def _to(self, dtype):
         return storage
 
     def double(self):
-        """Casts this storage to double type"""
+        """Casts this storage to double type."""
         _warn_typed_storage_removal()
         return self._to(torch.double)
 
     def float(self):
-        """Casts this storage to float type"""
+        """Casts this storage to float type."""
         _warn_typed_storage_removal()
         return self._to(torch.float)
 
     def half(self):
-        """Casts this storage to half type"""
+        """Casts this storage to half type."""
         _warn_typed_storage_removal()
         return self._to(torch.half)
 
     def long(self):
-        """Casts this storage to long type"""
+        """Casts this storage to long type."""
         _warn_typed_storage_removal()
         return self._to(torch.long)
 
     def int(self):
-        """Casts this storage to int type"""
+        """Casts this storage to int type."""
         _warn_typed_storage_removal()
         return self._to(torch.int)
 
     def short(self):
-        """Casts this storage to short type"""
+        """Casts this storage to short type."""
         _warn_typed_storage_removal()
         return self._to(torch.short)
 
     def char(self):
-        """Casts this storage to char type"""
+        """Casts this storage to char type."""
         _warn_typed_storage_removal()
         return self._to(torch.int8)
 
     def byte(self):
-        """Casts this storage to byte type"""
+        """Casts this storage to byte type."""
         _warn_typed_storage_removal()
         return self._to(torch.uint8)
 
     def bool(self):
-        """Casts this storage to bool type"""
+        """Casts this storage to bool type."""
         _warn_typed_storage_removal()
         return self._to(torch.bool)
 
     def bfloat16(self):
-        """Casts this storage to bfloat16 type"""
+        """Casts this storage to bfloat16 type."""
         _warn_typed_storage_removal()
         return self._to(torch.bfloat16)
 
     def complex_double(self):
-        """Casts this storage to complex double type"""
+        """Casts this storage to complex double type."""
         _warn_typed_storage_removal()
         return self._to(torch.cdouble)
 
     def complex_float(self):
-        """Casts this storage to complex float type"""
+        """Casts this storage to complex float type."""
         _warn_typed_storage_removal()
         return self._to(torch.cfloat)
 
     @classmethod
     def from_file(cls, filename, shared, size):
         """
-        from_file(filename, shared=False, size=0) -> Storage
+        from_file(filename, shared=False, size=0) -> Storage.
```

Comment:
I wonder if this change prevent the proper doc rendering - see https://pytorch.org/docs/stable/storage.html#torch.TypedStorage.from_file

We can verify after the merge conflict resolved.

Reply:
I'll remove the period since the signature should be maintained